components/chef-ui-library/src/molecules/chef-phat-radio/chef-phat-radio.tsx

```diff
@@ -67,4 +75,10 @@ export class ChefPhatRadio {
     return options;
   }
 
+  handleKeypress = (event) => {
+    if (event.key === 'Enter') {
+      this.handleClick(event);
+    }
+  }
```

Comment:
Stencil wouldn't allow me to add the click listening function straight to the keypress event listener, so I had to loop it around.

Reply:
Any reason to format this way instead of
```ts
handleKeypress(event) {
   ...
}
```

Reply:
absolutely not - thanks for the catch!